proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress.md

```diff
@@ -168,14 +179,13 @@ Due to the complexity of the flow, the developer must do the implementation care
 This proposal requires a major version change.
 Since new parameters are added, Core, iOS, Java Suite, RPC, Protocol, and HMI are affected.
 
-
 ## Alternatives considered
 
-The flow in the Proposed solution section is complicated, so the author has also considered the simplified solution below.
+While this alternative was considered, the Steering Committee determined this flow does not account for previous versions of the protocol, and therefore is not viable as the primary solution.
 
 ### Change the app registration flow
 
-![registeringApp_flow2](../assets/proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress/registeringApp_flow2.png)
+![registeringApp_flow1](../assets/proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress/registeringApp_flow1.png)
```

Comment:
This image is for the alternative solution, correct?  The Steering Committee didn't request any revisions to the alternative solution.  Can you please advise why you've changed this image?

Reply:
Hi, @theresalech -san.
Thank you for your prompt reply.
Yes, it is correct.
I only changed the file name about the proposal documentation and image file.
Thank you.

Reply:
@Akihiro-Miyazaki, I'm having trouble following if the flow depicted in the image is for the proposed solution or the solution described in the `Alternatives considered` section.  Regardless, this image change was not part of the Steering Committee's [agreed upon revisions](https://github.com/smartdevicelink/sdl_evolution/issues/941#issuecomment-648864859) and so should not be included in this PR.  Please revert to the image used previously, and if you would like a new image to be included in the proposal, you can comment on the review issue with the suggested change when the revised proposal is brought back into review.  Please let me know if you have any questions about this.  Thank you!

Reply:
@theresalech -san,
I modified a proposal documentation.
I also replaced the image files.
And I made a commit them.
Thank you.

Reply:
@Akihiro-Miyazaki , the image file should **not** be replaced in this PR, as that was not requested by the Steering Committee.  Please revert to the image used previously (registeringApp_flow2.png).  Thank you.

Reply:
@theresalech -san,
Thank you for your response.
What I did only was to replace (revert) the image file to previous one.
If this comment was wrong, could you please teach me how to revert?
Thank you in advance for your help.

Reply:
@Akihiro-Miyazaki I've confirmed the new image is no longer present in the `Alternatives Considered` section.  Thanks!